src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -121,14 +121,29 @@ public void systemLoadsAllergiesPage() {
 		assertTrue(textExists("Allergies"));
 	}
 
-	@When("a user clicks Add Known Allergy button")
+	@And("a user clicks Add Known Allergy for the first allergy")
 	public void addKnownAllergy() {
 		addOrEditAllergyPage = allergyPage.clickOnAddNewAllergy();
 		addOrEditAllergyPage.enterDrug(DRUG_NAME);
 		addOrEditAllergyPage.drugId();
 		addOrEditAllergyPage.enterReaction(REACTION);
 		addOrEditAllergyPage.reactionId();
 		addOrEditAllergyPage.addAllergyNote("The effect is severe");
+		addOrEditAllergyPage.clickOnSaveAllergy();
+	}
+
+	@And("a user clicks Add Known Allergy for the second allergy")
+	public void addAllergy() {
+		addOrEditAllergyPage = allergyPage.clickOnAddNewAllergy();
+	}
+
+	@And("the user selects an allergy")
+	public void selectAllergy() {
+		addOrEditAllergyPage.enterDrug("Codeine");
```

Comment:
How about making this globably something like this
```suggestion
		addOrEditAllergyPage.enterDrug(DRUG_NAME); 
```